src/Animals.java

```diff
@@ -0,0 +1,31 @@
+public class Animals {
+    String Name;
+    int RunLimit;
+    int SmimLimit;
+
+  Animals (int RunLimit, int SmimLimit){
+         this.RunLimit = RunLimit;
+         this.SmimLimit = SmimLimit;
+    }
+    //abstract void Swim(int dist);
+    public void Swim(int dist){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.